.gitignore

```diff
@@ -33,3 +33,6 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+coverage/*
+!coverage/index.html
```

Comment:
Why add exception for the `coverage/index.html`? It's an unusual thing to do, I'm curious on the reasoning behind it :)

Reply:
@mtrojanowska let's remove it as we agreed to do so on our phone conversation